app/components/municipality-select.hbs

```diff
@@ -0,0 +1,14 @@
+<PowerSelect
+  @allowClear={{true}}
+  @searchEnabled={{true}}
+  @loadingMessage="Aan het laden..."
+  @noMatchesMessage="Geen resultaten"
+  @searchMessage="Typ om te zoeken"
+  @options={{this.municipalities}}
```

Comment:
This is the TaskInstance we set earlier. If the promise is running, PowerSelect will show the loadingMessage when opened. Once the promise resolves the options will be available.